node/overseer/overseer-gen/README.md

```diff
@@ -76,3 +76,12 @@ subsystems must be initialized only once (another compile time check) or be _rep
 a special setter like method `replace_<subsystem>`.
 
 A task spawner and subsystem context are required to be defined with `SpawnNamed` and respectively `SubsystemContext` implemented.
+
+## Debugging
+
+As always, debugging is notoriously annoying with bugged proc-macros.
+
+Therefor [`expander`](https://github.com/drahnr/expander) is employed to yield better
```

Comment:
```suggestion
Therefore [`expander`](https://github.com/drahnr/expander) is employed to yield better
```

Reply:
As far as I know, it should be `Therefore, ` even :)

Reply:
Therefor is also valid, but archaic